tensorboard/plugins/graph/keras_util.py

```diff
@@ -92,89 +121,171 @@ def _is_model(layer):
   return layer.get('config').get('layers') is not None
```

Comment:
Unrelated, but this `get`-chaining isn’t correct, because `None.get` is
an error:

```
>>> {}.get('config').get('layers')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'NoneType' object has no attribute 'get'
```

Presumably you want `layer.get('config', {}).get('layers')`.